app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_all_numbers_are_zero() -> None:
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_first_14_years() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_first_24_year() -> None:
+    assert get_human_age(24, 24) == [2, 2]
+
+
+def test_first_28_year() -> None:
+    assert get_human_age(28, 28) == [3, 2]
```

Comment:
Make sure that you use `@pytest.mark.parametrize` to avoid code duplication and test all cases mentioned in the checklist including tests against raised errors.